docs/quickstart.md

```diff
@@ -1,197 +1,200 @@
+In this quickstart, we will do the following automatically with DevStream:
 
-Note : `dtm` currently doesn't support Windows yet.
+- create a GitHub repository with Golang web app scaffolding;
+- set up GitHub Actions workflow for the Golang app we created, which contains test and build stages for our Go web app.
 
-For Linux/Macos users:
+## 1 Download
 
-- rename the downloaded binary to `dtm` and move it to your PATH (e.g.: `mv dtm /usr/local/bin/`)
-- grant `dtm` executable permission (e.g.: `chmod a+x dtm`)
-
-## 2 Prepare a Config File
-
-Before you start: for an example of DevStream config, see [examples/quickstart.yaml](https://github.com/devstream-io/devstream/blob/main/examples/quickstart.yaml). 
-Remember to open this configuration file, modify all FULL_UPPER_CASE_STRINGS (like YOUR_GITHUB_USERNAME, for example) in it to your own. Pay attention to the meaning of each item to ensure that it is what you want. For other plugins, checkout the "Plugins" section in our [doc](https://docs.devstream.io) for detailed usage.
-
-Download the [examples/quickstart.yaml](https://raw.githubusercontent.com/devstream-io/devstream/main/examples/quickstart.yaml) to your working directory:
+In your working directory, run:
 
 ```shell
-curl -o quickstart.yaml https://raw.githubusercontent.com/devstream-io/devstream/main/examples/quickstart.yaml
+sh -c "$(curl -fsSL https://raw.githubusercontent.com/devstream-io/devstream/main/hack/quick-start/quickstart.sh)"
 ```
 
-Then modify the file accordingly.
+This will download the `dtm` binary and a `quickstart.yaml` config file to your working directory, and grant the binary execution permission.
 
-For example, my GitHub username is "IronCore864", and my Dockerhub username is "ironcore864", then I can run:
+> Optional: you can then move `dtm` to a place which is in your PATH. For example: `mv dtm /usr/local/bin/`.
 
-```shell
-sed -i.bak "s/YOUR_GITHUB_USERNAME_CASE_SENSITIVE/IronCore864/g" quickstart.yaml
+## 2 Configuration
 
-sed -i.bak "s/YOUR_DOCKER_USERNAME/ironcore864/g" quickstart.yaml
-```
+As aforementioned, we will handle GitHub repo scaffolding and CI workflows in GitHub Actions, so, we will need the following environment variables to be set:
 
-> This config file uses two plugins, one will create a GitHub repository and bootstrap it into a Golang web app, and the other will create GitHub Actions workflow for it.
+- GITHUB_USER
+- GITHUB_TOKEN
+- DOCKERHUB_USERNAME
 
-The two plugins, [githubactions-golang](./plugins/githubactions-golang.md) &
-[github-repo-scaffolding-golang](./plugins/github-repo-scaffolding-golang.md), require an environment variable to work, so let's set it:
+For example, if my GitHub username is "IronCore864" and my Docker Hub username is "ironcore864", I run the following commands:
 
 ```shell
+export GITHUB_USER="IronCore864"
 export GITHUB_TOKEN="YOUR_GITHUB_TOKEN_HERE"
```

Comment:
How about leave a "fake token" here? just like:

12313uijsioj9uwqr2jiorjfklsjfljsd

Reply:
I think it's better to use placeholders (we can use them for all env vars to be consistent.)

Using a token-like string might still trigger static code scanning tools (which we don't have yet, though,) so it's not a best practice.